src/main/java/com/adaptavist/tm4j/jenkins/extensions/configuration/Tm4jGlobalConfiguration.java

```diff
@@ -46,47 +48,67 @@ public boolean configure(StaplerRequest request, JSONObject formData) throws For
         Permissions.checkAdminPermission();
         request.bindParameters(this);
         Object formJiraInstances = formData.get("jiraInstances");
```

Comment:
jiraInstances is being used multiple times can be extracted to a constant (to be less error prone)